portfolio/src/main/webapp/style.css

```diff
@@ -106,6 +114,31 @@ td p {
   color: rgb(61,95,121);
 }
 
+#hobbies-div {
+  color: white;
+}
+
+.hobbies-page-a {
```

Comment:
the name 'page-a' is not very meaningful. Can you rename it?

And it seems it is not following alphabetic order.

Reply:
Will do. 
With alphabetization, color and text-decoration should be in the right order, do you mean that the entire block should be placed somewhere else?